src/test/java/org/mule/extensions/jms/test/util/ActiveMQConnectionFactoryUtilTestCase.java

```diff
@@ -18,6 +20,23 @@ public void testVersionValid() {
     assertFalse(ActiveMQConnectionFactoryUtil.isVerifyHostnameValidVersion("5.14.5"));
     assertFalse(ActiveMQConnectionFactoryUtil.isVerifyHostnameValidVersion("4.10.5"));
     assertTrue(ActiveMQConnectionFactoryUtil.isVerifyHostnameValidVersion("5.15.6"));
+    assertTrue(ActiveMQConnectionFactoryUtil.isVerifyHostnameValidVersion("5.18.3"));
     assertTrue(ActiveMQConnectionFactoryUtil.isVerifyHostnameValidVersion("6.0.0"));
   }
+
```

Comment:
What about testing the brokerUrlFormat with verifyHostName=true?

Reply:
done